tests/installation/ipxe_install.pm

```diff
@@ -88,41 +92,7 @@ sub set_bootscript {
         $install .= "?device=$interface ifcfg=$interface=dhcp4 ";
     }
 
-    my $cmdline_extra;
-    $cmdline_extra .= " regurl=$regurl " if ($regurl and !is_usb_boot);
-    $cmdline_extra .= " console=$console " if $console;
-    $cmdline_extra .= " root=/dev/ram0 initrd=initrd " if (check_var('IPXE_UEFI', '1'));
-    $cmdline_extra .= " textmode=1 " if get_var('IPXE_UEFI') or check_var('VIDEOMODE', 'text');
-    $cmdline_extra .= " self_update=0 " if (check_var("INSTALLER_NO_SELF_UPDATE", 1));
-
-    # Support passing EXTRA_PXE_CMDLINE to bootscripts
-    $cmdline_extra .= get_var('EXTRA_PXE_CMDLINE') . ' ' if get_var('EXTRA_PXE_CMDLINE');
-
-    if ($autoyast ne '') {
-        $cmdline_extra .= " autoyast=$autoyast sshd=1 sshpassword=$testapi::password ";
-    } else {
-        $cmdline_extra .= " ssh=1 sshpassword=$testapi::password ";
-        $cmdline_extra .= " vnc=1 VNCPassword=$testapi::password " unless check_var('VIDEOMODE', 'text');
-    }
-    $cmdline_extra .= " plymouth.enable=0 ";
-
-    $cmdline_extra .= " video=1024x768 vt.color=0x07 " if check_var('VIDEOMODE', 'text');
-    # Support either IPXE_CONSOLE=ttyS1,115200 or SERIALDEV=ttyS1
-    my $serial_dev;
-    if (get_var('IPXE_CONSOLE')) {
-        get_var('IPXE_CONSOLE') =~ /^(\w+)/;
-        $serial_dev = $1;
-    }
-    else {
-        $serial_dev = get_var('SERIALDEV', 'ttyS1');
-        $cmdline_extra .= " console=$serial_dev,115200 ";
-    }
-
-    # Extra options for virtualization tests with ipmi backend
-    $cmdline_extra .= " Y2DEBUG=1 linuxrc.log=/dev/$serial_dev linuxrc.core=/dev/$serial_dev linuxrc.debug=4,trace ";
-    $cmdline_extra .= " reboot_timeout=" . get_var('REBOOT_TIMEOUT', 0) . ' '
-      unless (is_leap('<15.2') || is_sle('<15-SP2'));
-    $cmdline_extra .= get_var('EXTRABOOTPARAMS', '');
+    my $cmdline_extra = set_bootscript_cmdline_extra();
```

Comment:
Why it was moved to function? It is used only once.

Reply:
Also is hard to what has changed as it was moved to different place.

Reply:
It is not in a different place. The new subroutine ```set_bootscript_cmdline_extra``` is in the same module. I think it would be better to wrap these things up, otherwise the ``set_bootscript``` will continue to expand in a way that does not look tidy. It is already a very lengthy subroutine.